tests/mul.s

```diff
@@ -0,0 +1,4 @@
+mul $3 $2 #3
```

Comment:
You should consider special cases like the multiplication of two negative numbers.